sjsonnet/test/src-jvm/sjsonnet/ErrorTests.scala

```diff
@@ -288,20 +288,12 @@ object ErrorTests extends TestSuite{
          |    at .(sjsonnet/test/resources/imports/error.too_many_arg.jsonnet:3:6)
          |""".stripMargin
     )
-    //    test("overflow") - check(
```

Comment:
@lihaoyi-databricks was there a reason to have these commented out? If yes, we should add a `TODO:` so we know why.

Reply:
Honestly I don't remember, but I assume it's because I never managed to get them passing. Looking at https://github.com/databricks/sjsonnet/commit/c874391d3f81915001e2d2eae26f54cc33a2c998 which introduced the other tests in this suite, it appears these tests were introduced already-commented-out, so we could probably just delete them